Adyen/Integration/02_ManageAdyenConnection.md

```diff
@@ -0,0 +1,161 @@
+# Manage Adyen connections
+
+In the following, the procedures to create, edit, configure, enable, or disable Adyen&trade; connections in the *Payments* module are described.  
+
+Note the following:   
+- If you have more than one merchant account in Adyen&trade; that you want to manage with Actindo, you must create a connection for each.   
+- Adyen&trade; offers the possibility to work with two separate environments, a staging environment for testing purposes and a live environment for production. You must configure the connection for each environment separately.
+- It is recommended that you manage the Adyen&trade; settings in parallel, so that you can enter the connection data alternately. Note that you can no longer copy the credentials after you have saved the Adyen&trade; or Payments settings. For detailed information, see [Manage Adyen settings](01_ManageAdyenSettings.md).
+
+
+## Create Adyen connection
+Create a connection to the Adyen&trade; payment service provider.
+
+#### Prerequisites
+
+- You have an Adyen&trade; customer account to log in to Adyen&trade;.
+- You are logged in to Adyen&trade;.
+- You are assigned to the required user roles in Adyen&trade;.
+- In case you are creating connection to an Adyen™ live account, you have the API endpoints for both the *Checkout API* endpoint and the *Payment API* endpoint at hand.
+- In the *Actindo Core1 Platform*, you have permission to create connections in the *Payments* module.
+
+#### Procedure
+
+ *Payments > Settings > Tab CONNECTIONS*
+ 
+ ![Connections](../../Assets/Screenshots/Payments/Settings/Settings.png "[Connections]")
+
+1.  Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Create Connection* view is displayed.  
+
+    ![Create connection](../../Assets/Screenshots/Adyen/Integration/CreateConnection.png "[Create connection]")
+
+2.  Enter a name for the connection in the *Name* field.
+
+3.  Click the *Driver* drop-down list and select *Adyen*. 
+   The *Credentials* section is displayed.
+
+    ![Credentials](../../Assets/Screenshots/Payments/Settings/CreateConnectionCredentials.png "[Credentials]")
+
+4. Enter the API endpoints and credentials. To do this, follow these steps:
+    > [Info] There are some standard access points for a testing environment. For your productive/live environment, Adyen&trade; provides you with individual API endpoints. 
+
+    -  Enter the endpoint of the Checkout API in the *Checkout API endpoint* field. 
```

Comment:
"endpoint of the checkout API" oder "checkout API endpoint" 
Ich meine, Groß-/Kleinchreibung und Reihenfolge zur Konsistenz mit "payment API endpoint" (s. weiter unten)